README.md

```diff
@@ -1,39 +1,43 @@
 <p align="center">
-  <a href="https://www.gatsbyjs.com/?utm_source=starter&utm_medium=readme&utm_campaign=minimal-starter">
-    <img alt="Gatsby" src="https://www.gatsbyjs.com/Gatsby-Monogram.svg" width="60" />
-  </a>
-</p>
+    <p align="center">
+      <a href="https://github.com/process-analytics/website">
+        <img alt="Process Analytics Logo" src="public/icons/logo_64x64.ico" width="64" />
```

Comment:
We should remove the public folder from the repo. It is generated. 
We should move assets to the src folder

Reply:
I change the icons in another PR. I'll change the folder in this one.